test/e2e/set/scyllacluster/scyllacluster_rebootstrap.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2021 ScyllaDB
+
+package scyllacluster
+
+import (
+	"context"
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	gt "github.com/onsi/ginkgo/extensions/table"
+	o "github.com/onsi/gomega"
+	scyllaclusterfixture "github.com/scylladb/scylla-operator/test/e2e/fixture/scyllacluster"
+	"github.com/scylladb/scylla-operator/test/e2e/framework"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/utils/pointer"
+)
+
+var _ = g.Describe("ScyllaCluster", func() {
+	defer g.GinkgoRecover()
+
+	f := framework.NewFramework("scyllacluster")
+
+	describe := func(storageClassName string) string {
+		return fmt.Sprintf("with StorageClass: %s", storageClassName)
+	}
+
+	gt.DescribeTable("should re-bootstrap from old PVCs",
+		func(storageClassName string) {
+			var err error
+
+			ctx, cancel := context.WithTimeout(context.Background(), testTimout)
+			defer cancel()
+
+			framework.By("Verifying StorageClass' presence")
+			storageClassList, err := f.KubeAdminClient().StorageV1().StorageClasses().List(ctx, metav1.ListOptions{})
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			storageClassExists := false
+			for _, storageClass := range storageClassList.Items {
+				if storageClass.Name == storageClassName {
+					storageClassExists = true
+					break
+				}
+			}
+			if !storageClassExists {
+				g.Skip("StorageClass missing, skipping test")
+			}
```

Comment:
you know name you're looking for, why not use Get instead of List and check if err is NotFound?

Reply:
good point, fixed